test-suite/mk-tls-test-certs.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+#
+# test certificates and keys
+#
+
+if test -x /usr/bin/certtool; then
+
+echo "
+organization = \"Example.Org\"
+cn = \"Test CA. Do not Trust\"
+dc = \"example.org\"
+expiration_days = 700
+ca
+cert_signing_key
+crl_signing_key
+tls_www_server
+" >example.org-ca
+
+#
+# self-signed root CA
+#
+/usr/bin/certtool --generate-privkey --rsa --outfile ca-root-rsa.pkey
```

Comment:
The script should not ignore certtool (and other) errors. The script should fail if one of its steps fails.

Reply:
Added certtool detection (and openssl's equivalent tool) to configure.ac.

I'm not sure what the config file for openssl should contain for each of the cert creation steps to avoid manual question/answers. Any assistance welcome.

Reply:
> I'm not sure what the config file for openssl should contain for each of the cert creation steps to avoid manual question/answers.

I emailed you a very rough script (and an OpenSSL configuration file sketch) based on what we have used for some Factory tests. Adjust as needed.